bql/planner/data_access_test.go

```diff
@@ -311,61 +311,65 @@ func TestDataAccessBasicBindings(t *testing.T) {
 		PBinding: "?p",
 		OBinding: "?o",
 	}
+
 	testTable := []struct {
-		t  string
-		sc *table.Cell
-		pc *table.Cell
-		oc *table.Cell
+		t        string
+		sBinding *table.Cell
+		pBinding *table.Cell
+		oBinding *table.Cell
 	}{
 		{
-			t:  n.String() + "\t" + p.String() + "\t" + n.String(),
-			sc: &table.Cell{N: n},
-			pc: &table.Cell{P: p},
-			oc: &table.Cell{N: n},
+			t:        n.String() + "\t" + p.String() + "\t" + n.String(),
```

Comment:
You can do this by using the `fmt` package in a more concise way:
`fmt.Sprintf("%s\t%s\t%s", n, p, n)`
`%s` should call the `String` method for you :D

Please do that here and below :)

Reply:
Thank you for the comment Tati, I have just done so. 😀